packages/core/src/internal-components/visual-checkbox/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export * from './visual-checkbox.element.js';
```

Comment:
I don't think this approach will work after the Lit 2 upgrade PR. IIRC, rollup has a problem with barrel imports referencing barrel imports. Barrel Inception?

Maybe we should remove this file? What say you, @coryrylan ?

Reply:
This should be fine because its the index of the entrypoint `visual-checkbox`